Missing golint dependency on make

Using v0.7.8 source code and running `make` fails with:
`make: golint: No such file or directory`

Full log:
```
go get golang.org/x/lint/golint
go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.33.0
go get github.com/axw/gocov/...
go get github.com/AlekSi/gocov-xml
go get github.com/tebeka/go2xunit
▶ running gofmt...
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/sirupsen/logrus v1.7.0
go: downloading golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
go: downloading github.com/urfave/cli v1.22.4
go: downloading github.com/johntdyer/slackrus v0.0.0-20180518184837-f7aae3243a07
go: downloading github.com/docker/engine v17.12.0-ce-rc1.0.20190717161051-705d9623b7c1+incompatible
go: downloading github.com/golangci/golangci-lint v1.33.0
go: downloading github.com/AlekSi/gocov-xml v0.0.0-20190121064608-3a14fb1c4737
go: downloading github.com/axw/gocov v1.0.0
go: downloading github.com/tebeka/go2xunit v1.4.10
go: downloading github.com/axw/gocov v1.0.0
go: downloading golang.org/x/tools v0.0.0-20201013201025-64a9e34f3752
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
# github.com/AlekSi/gocov-xml
loadinternal: cannot find runtime/cgo
go: downloading golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
go: downloading golang.org/x/tools v0.0.0-20201013201025-64a9e34f3752
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
# github.com/tebeka/go2xunit
loadinternal: cannot find runtime/cgo
go: downloading github.com/fatih/color v1.10.0
go: downloading github.com/gofrs/flock v0.8.0
go: downloading github.com/spf13/cobra v1.1.1
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/spf13/viper v1.7.1
go: downloading golang.org/x/tools v0.0.0-20201013201025-64a9e34f3752
go: downloading github.com/go-critic/go-critic v0.5.2
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/mattn/go-colorable v0.1.8
go: downloading github.com/stretchr/testify v1.6.1
go: downloading github.com/go-xmlfmt/xmlfmt v0.0.0-20191208150333-d5b6f63a941b
go: downloading github.com/golangci/revgrep v0.0.0-20180526074752-d9c87f5ffaf0
go: downloading golang.org/x/sys v0.0.0-20201009025420-dfb3f7c4e634
go: downloading 4d63.com/gochecknoglobals v0.0.0-20201008074935-acfc0b28355a
go: downloading github.com/Djarvur/go-err113 v0.0.0-20200511133814-5174e21577d5
go: downloading github.com/OpenPeeDeeP/depguard v1.0.1
go: downloading github.com/bombsimon/wsl/v3 v3.1.0
go: downloading github.com/daixiang0/gci v0.2.4
go: downloading github.com/denis-tingajkin/go-header v0.3.1
go: downloading github.com/golangci/check v0.0.0-20180506172741-cfe4005ccda2
go: downloading github.com/golangci/dupl v0.0.0-20180902072040-3e9179ac440a
go: downloading github.com/golangci/errcheck v0.0.0-20181223084120-ef45e06d44b6
go: downloading github.com/golangci/go-misc v0.0.0-20180628070357-927a3d87b613
go: downloading github.com/golangci/gocyclo v0.0.0-20180528144436-0a533e8fa43d
go: downloading github.com/docker/go-connections v0.4.0
go: downloading github.com/golangci/gofmt v0.0.0-20190930125516-244bba706f1a
go: downloading golang.org/x/sys v0.0.0-20201009025420-dfb3f7c4e634
go: downloading github.com/johntdyer/slack-go v0.0.0-20180213144715-95fac1160b22
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.0
go: downloading github.com/docker/go-units v0.4.0
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading github.com/stretchr/objx v0.3.0
go: downloading github.com/golangci/ineffassign v0.0.0-20190609212857-42439a7714cc
go: downloading github.com/golangci/lint-1 v0.0.0-20191013205115-297bf364a8e0
go: downloading github.com/golangci/maligned v0.0.0-20180506175553-b1d89398deca
go: downloading github.com/opencontainers/image-spec v1.0.1
go: downloading github.com/golangci/misspell v0.0.0-20180809174111-950f5d19e770
go: downloading github.com/docker/distribution v2.7.1+incompatible
go: downloading google.golang.org/grpc v1.30.0
go: downloading github.com/opencontainers/go-digest v1.0.0
go: downloading gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
go: downloading github.com/gogo/protobuf v1.3.1
go: downloading github.com/golangci/prealloc v0.0.0-20180630174525-215b22d4de21
go: downloading github.com/golangci/unconvert v0.0.0-20180507085042-28b1c447d1f4
go: downloading github.com/jgautheron/goconst v0.0.0-20201117150253-ccae5bf973f3
go: downloading github.com/jingyugao/rowserrcheck v0.0.0-20191204022205-72ab7603b68a
go: downloading github.com/jirfag/go-printf-func-name v0.0.0-20191110105641-45db9963cdd3
go: downloading github.com/kunwardeep/paralleltest v1.0.2
go: downloading github.com/kyoh86/exportloopref v0.1.8
go: downloading github.com/russross/blackfriday/v2 v2.0.1
go: downloading github.com/maratori/testpackage v1.0.1
go: downloading github.com/matoous/godox v0.0.0-20190911065817-5d6d842e92eb
go: downloading github.com/mbilski/exhaustivestruct v1.1.0
go: downloading github.com/moricho/tparallel v0.2.1
go: downloading github.com/nakabonne/nestif v0.3.0
go: downloading github.com/nishanths/exhaustive v0.1.0
go: downloading github.com/polyfloyd/go-errorlint v0.0.0-20201006195004-351e25ade6e3
go: downloading github.com/ryancurrah/gomodguard v1.1.0
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
go: downloading github.com/ryanrolds/sqlclosecheck v0.3.0
go: downloading github.com/securego/gosec/v2 v2.5.0
go: downloading github.com/shazow/go-diff v0.0.0-20160112020656-b6b7b6733b8c
go: downloading github.com/sonatard/noctx v0.0.1
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
go: downloading github.com/sourcegraph/go-diff v0.6.1
go: downloading github.com/ssgreg/nlreturn/v2 v2.1.0
go: downloading github.com/tdakkota/asciicheck v0.0.0-20200416190851-d7f85be797a2
go: downloading github.com/tetafro/godot v1.3.0
go: downloading golang.org/x/net v0.0.0-20200822124328-c89045814202
go: downloading github.com/timakin/bodyclose v0.0.0-20190930140734-f7f2e9bca95e
go: downloading github.com/tomarrell/wrapcheck v0.0.0-20200807122107-df9e8bcb914d
go: downloading github.com/tommy-muehle/go-mnd v1.3.1-0.20200224220436-e6f9a994e8fa
go: downloading github.com/ultraware/funlen v0.0.3
go: downloading github.com/ultraware/whitespace v0.0.4
go: downloading github.com/uudashr/gocognit v1.0.1
go: downloading honnef.co/go/tools v0.0.1-2020.1.6
go: downloading mvdan.cc/gofumpt v0.0.0-20200802201014-ab5a8192947d
go: downloading mvdan.cc/interfacer v0.0.0-20180901003855-c20040233aed
go: downloading mvdan.cc/unparam v0.0.0-20200501210554-b37ab49443f7
go: downloading github.com/fsnotify/fsnotify v1.4.9
# github.com/axw/gocov/gocov
loadinternal: cannot find runtime/cgo
# golang.org/x/lint/golint
loadinternal: cannot find runtime/cgo
go get: upgraded golang.org/x/lint v0.0.0-20200302205851-738671d3881b => v0.0.0-20210508222113-6edffad5e616
go: downloading google.golang.org/genproto v0.0.0-20200715011427-11fb19a81f2c
go: downloading github.com/golang/protobuf v1.4.2
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading github.com/magiconair/properties v1.8.1
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/pelletier/go-toml v1.2.0
go: downloading github.com/spf13/afero v1.1.2
go: downloading github.com/spf13/cast v1.3.0
go: downloading github.com/spf13/jwalterweatherman v1.0.0
go: downloading github.com/subosito/gotenv v1.2.0
go: downloading gopkg.in/ini.v1 v1.51.0
go: downloading gopkg.in/yaml.v2 v2.3.0
go: downloading github.com/mattn/go-isatty v0.0.12
go: downloading github.com/go-toolsmith/astfmt v1.0.0
go: downloading github.com/go-toolsmith/astcast v1.0.0
go: downloading github.com/go-toolsmith/astcopy v1.0.0
go: downloading github.com/go-toolsmith/astequal v1.0.0
go: downloading github.com/go-toolsmith/astp v1.0.0
go: downloading github.com/go-toolsmith/strparse v1.0.0
go: downloading github.com/go-toolsmith/typep v1.0.2
go: downloading github.com/quasilyte/go-ruleguard v0.2.0
go: downloading github.com/quasilyte/regex/syntax v0.0.0-20200407221936-30656e2c4a95
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: downloading golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
go: downloading github.com/gobwas/glob v0.2.3
go: downloading github.com/kisielk/gotool v1.0.0
go: downloading golang.org/x/text v0.3.3
go: downloading github.com/gostaticanalysis/analysisutil v0.1.0
go: downloading github.com/Masterminds/semver v1.5.0
go: downloading github.com/phayes/checkstyle v0.0.0-20170904204023-bfd46e6a821d
go: downloading golang.org/x/mod v0.3.0
go: downloading github.com/nbutton23/zxcvbn-go v0.0.0-20180912185939-ae427f1e4c1d
go: downloading github.com/google/go-cmp v0.5.2
go: downloading mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b
go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
go: downloading google.golang.org/protobuf v1.25.0
go: downloading github.com/BurntSushi/toml v0.3.1
go: downloading github.com/gostaticanalysis/comment v1.3.0
▶ running golint...
▶ running tests...
make: golint: No such file or directory
make: *** [Makefile:140: lint] Error 127
make: *** Waiting for unfinished jobs....
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
# github.com/golangci/golangci-lint/cmd/golangci-lint
loadinternal: cannot find runtime/cgo
```
FWIW I'm using
```
$ go version
go version go1.17 linux/amd64
```

--- a/Makefile
+++ b/Makefile
@@ -66,19 +66,19 @@ release: clean ; $(info $(M) building binaries for multiple os/arch...) @ ## Bui
 setup-tools: setup-golint setup-golangci-lint setup-gocov setup-gocov-xml setup-go2xunit
 
 setup-golint:
-	$(GO) get golang.org/x/lint/golint
+	$(GO) install golang.org/x/lint/golint
 setup-golangci-lint:
-	$(GO) get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.33.0
+	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.33.0
 setup-gocov:
-	$(GO) get github.com/axw/gocov/...
+	$(GO) install github.com/axw/gocov/...
 setup-gocov-xml:
-	$(GO) get github.com/AlekSi/gocov-xml
+	$(GO) install github.com/AlekSi/gocov-xml
 setup-go2xunit:
-	$(GO) get github.com/tebeka/go2xunit
+	$(GO) install github.com/tebeka/go2xunit
 setup-mockery:
-	$(GO) get github.com/vektra/mockery/v2/
+	$(GO) install github.com/vektra/mockery/v2/
 setup-ghr:
-	$(GO) get github.com/tcnksm/ghr
+	$(GO) install github.com/tcnksm/ghr
 
 GOLINT=golint
 GOCOV=gocov

